developer-docs-site/docs/concepts/coin-and-token/aptos-token.md

```diff
@@ -155,14 +161,87 @@ The token-related data are stored at both creator’s account and owner’s acco
 | Royalty | Specifies the denominator and numerator for calculating the royalty fee. It also has the payee account address for depositing the royalty. |
 | `PropertyValue` | Contains both value of a property and type of property. |
 
-### Resource stored at the owner’s address
+#### Resource stored at the owner’s address
 
 | Field | Description |
 | --- | --- |
 | `TokenStore` | The main struct for storing the token owned by this address. It maps `TokenId` to the actual token. |
 | `Token` | The amount is the number of tokens. |
 | `TokenId` | `TokenDataId` points to the metadata of this token. The `property_version` represents a token with mutated `PropertyMap` from `default_properties` in the `TokenData`. |
 
+### Field descriptions
+
+The following tables contain descriptions and examples on each token field.
+
+#### Coin module key struct
```

Comment:
This doc is only for describing token.move. We can remove the coin stuff from here

Reply:
Shall I put this information on the Coin page?